developer/src/kmcmplib/src/Compiler.cpp

```diff
@@ -2226,8 +2225,10 @@ KMX_DWORD GetXStringImpl(PKMX_WCHAR tstr, PFILE_KEYBOARD fk, PKMX_WCHAR str, KMX
 
           kmcmp::CheckStoreUsage(fk, i, TRUE, FALSE, FALSE);
 
-          r = u16tok(NULL, p_sep_com, &context);  // I3481
-          if (!r || !*r || !isIntegerWstring(r) || atoiW(r) < 1) return KmnCompilerMessages::ERROR_InvalidIndex;
+          r = u16tok(NULL, ',', &context);  // I3481
```

Comment:
Technically, this should not allow another comma at this point? We just want the rest of the string?

`index(a,1,2)` should fail compile, but I don't think it will at this point

Reply:
I was too tempted by the minimal change ... corrected and matching test added.